app/main/views/users.py

```diff
@@ -24,19 +24,19 @@ def find_user_by_email_address():
     if email_address:
         users = data_api_client.get_user(email_address=request.args.get("email_address"))
 
```

Comment:
Slightly outside the changed code here but we assign the `request.args email address` or `None` to the variable `email_address` earlier in the method then repeatedly reference it from the `request.args` anyway. Would be good to make this consistent.